### ISSUE TITLE
图片放到之后有黑色边框

### ISSUE THREAD

**Issue by szdenny2** [BigBadaboom/androidsvg#281]
![image](https://github.com/BigBadaboom/androidsvg/assets/18253729/275fba6f-6d94-456e-9f22-e087d9356459)

**Comment by BigBadaboom** [comment]
I am assuming your SVG does not have a dark stroke (border)?

Then I think you are seeing this because you are drawing the SVG shape 10 times (`NUM_BOUQUETS `) in exactly the same place.  The partially transparent anti-aliased pixels, at the edge of the SVG shape, are adding together making a dark halo around the shape.

If you remove the loop, so that you draw the image only once, then I think you will see that the dark line will disappear.

**Comment by BigBadaboom** [comment]
No further communication, so closing this ticket.

If you think this is still a bug, please let me know.
I also just realised that this could be the same bug as #289.

**Comment by BigBadaboom** [comment]
Marking this a duplicate of #289.
